src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs

```diff
@@ -39,7 +39,7 @@ internal static int[] GetCurrencyNegativePatterns(string localeName)
                     {
                         return (PlatformDetection.WindowsVersion < 10) ? new int[] { 3 } : new int[] { 6, 3 };
                     }
-                    if (PlatformDetection.ICUVersion.Major == 59)
+                    if (PlatformDetection.ICUVersion.Major == 59 || PlatformDetection.ICUVersion.Major == 58)
```

Comment:
Did the behavior change before these versions or after?  I'm wondering if it should be e.g.
```C#
if (PlatformDetection.ICUVersion.Major >= 58)
```
so that don't have to update this again when 60 comes out with the same behavior as 58 and 59.

Reply:
look at the else block 

```C#
                    else if (PlatformDetection.ICUVersion.Major > 59)
                    {
                        return new int[] { 1 };
                    }
```

That define the behavior after version 59.

Reply:
I see; that was hidden in the diff.  So we didn't hit this before because no platform we tested on had version 58 until now?

Reply:
right, it happens that Alpine image using ICU version 58.2.